include/erasure_code.h

```diff
@@ -222,6 +222,36 @@ void gf_vect_mad(int len, int vec, int vec_i, unsigned char *gftbls, unsigned ch
 void gf_vect_mad_base(int len, int vec, int vec_i, unsigned char *v, unsigned char *src,
 		      unsigned char *dest);
 
+// aarch64 only
```

Comment:
We don't want to add any new arch-specific exported functions.  Our goal is to eliminate all the arch-specific versions in the next major release so we don't want to add any new, non-backwards compatibility when they are removed.

Reply:
> We don't want to add any new arch-specific exported functions. Our goal is to eliminate all the arch-specific versions in the next major release so we don't want to add any new, non-backwards compatibility when they are removed.

remove arch-specific